extensions/shared/with-has-warning-is-interactive-class-names/style.scss

```diff
@@ -0,0 +1,12 @@
+.block-editor-block-list__layout
```

Comment:
Might be worth noting with a comment why these overrides exist. Not a blocker for merging.

Reply:
I see it's explained in https://github.com/Automattic/jetpack/pull/13019/files#diff-37c66df99b9d7cba3dc6798b7e53d938R11 but might be worth reiterating nevertheless.

Reply:
Added a note in https://github.com/Automattic/jetpack/pull/13019/commits/5f9b77aea9b8bb49ef98101338d566a1b2f1c760